tests/charts/test_basic_helm_chart.py

```diff
@@ -97,6 +97,78 @@ def test_basic_deployments(self):
                 "TEST-LABEL"
             ), f"Missing label TEST-LABEL on {k8s_name}. Current labels: {labels}"
 
+    def test_basic_deployment_with_standalone_scheduler(self):
+        k8s_objects = render_chart(
+            "TEST-BASIC",
+            values={
+                "chart": {
+                    'metadata': 'AA',
+                },
+                'labels': {"TEST-LABEL": "TEST-VALUE"},
+                "fullnameOverride": "TEST-BASIC",
+                "dagProcessor": {'enabled': True},
```

Comment:
If we keep it enabled by default, I can merge the assertions into the test_basic_deployment

Reply:
I think being disabled by default at this point makes sense.